templates/custom/media/media--image--teaser.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
+/**
+ * @file
+ * Theme override to display a media item.
+ *
+ * Available variables:
+ * - name: Name of the media.
+ * - content: Media content.
+ *
+ * @see template_preprocess_media()
+ *
+ * @ingroup themeable
+ */
+#}
+<a{{ attributes }} href="{{ file_url(content.field_media_image.0['#item'].entity.uri.value) }}" aria-label="image caption" aria-describedby="image-gallery__open-gallery">
```

Comment:
Aria-label should be content of figcaption. If there is no figcaption, don't add the aria-label attribute.

Aria-describedby should reference an element with a unique ID.
We could start adding some of these generic texts to the page template... They'll be used on virtually every page anyway.

Reply:
Agreed.